plugin/src/test/resources/com/google/errorprone/xplat/checker/testdata/JodaTimeObjectParamPositiveCases.java

```diff
@@ -0,0 +1,20 @@
+package com.google.errorprone.xplat.checker.testdata;
+
+import org.joda.time.DateTime;
+import org.joda.time.LocalDateTime;
+
+
+public class JodaTimeObjectParamPositiveCases {
+
+  public static void main(String[] args) {
+    Object ob = null;
+
+    // testing constructor with object param (not boxed long)
+    // BUG: Diagnostic contains: DateTime(java.lang.Object) is a banned constructor
+    DateTime time = new DateTime(ob);
```

Comment:
Although perhaps a trivial case, if you add another case like `new DateTime(null)`, would the checker be able to call it out?